wca-regulations.md

```diff
@@ -19,7 +19,7 @@ Original source of the WCA regulations: [www.worldcubeassociation.org/regulation
 WCA Regulations in [PDF format](link:pdf)
 
 ### Source
-Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/cubing/wca-documents).
+Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-documents).
```

Comment:
Lol, please grep for "contact/wrc" in the WCA Regulations translations.
It would be awesome if you could fix these too as part of thewca/wca-regulations-translations#41